### ISSUE TITLE
namedtuple with defaults:  Arguments missing

### ISSUE THREAD

**Issue by nschloe** [microsoft/pyright#1881]
The following is legal Python (3.7+), but pyright reports and error
```python
from collections import namedtuple


Info = namedtuple(
    "Info",
    ["a", "b", "c"],
    # last two default to None
    defaults=(None, None),
)

Info(3)
```
```
Arguments missing for parameters "b", "c" (reportGeneralTypeIssues)
```

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next release.

**Comment by erictraut** [comment]
This is fixed in pyright 1.1.141, which I just published. It will also be included in the next release of pylance.
